elastic/apm-server
# processor/otel: translate OpenTelemetry exceptions

## Motivation/summary

Add support for translating OpenTelemetry exception span events to errors, like we do with Jaeger error span logs. This includes parsing Java stack traces. We can add support for parsing stack traces from other languages later.

Also, stop adding HTTP *span* context to errors; only do this for transactions, to be consistent with Elastic APM agents.

## Checklist


## How to test these changes

1. Run APM Server
1. Instrument a Java application with OpenTelemetry, point it at APM Server
1. Provoke the application into throwing an exception
1. Check that the exception is recorded as an error in Elasticsearch, and that the stack trace is parsed and recorded as individual frames.

## Related issues

Closes #4712
